osbenchmark/workload/loader.py

```diff
@@ -586,6 +588,12 @@ def prepare_document_set(self, document_set, data_root):
                     else:
                         raise
         if document_set.support_file_offset_table:
+            if not document_set.source_url:
+                try:
+                    self.downloader.download(document_set.base_url, None, doc_path + '.offset', None)
+                except exceptions.DataError as e:
+                    if isinstance(e.cause, urllib.error.HTTPError) and (e.cause.code == 403 or e.cause.code == 404):
```

Comment:
for file not found ,shouldn't error code be just 404 ?

Reply:
If there is no read permission enabled on the directory, there will be a 403 error code for non-existent files or objects.

Reply:
Got it. My comment was more in regards to the log message, since it had only "file not found" message, I was wondering why 403 is also included here.  Thanks for clarifying